db/schema.rb

```diff
@@ -21,29 +29,44 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "nested_comments", force: :cascade do |t|
+    t.string "text"
```

Comment:
I don't think this needs the text. Nested Comments are just like the postTags relation.

Reply:
Why wouldn't it need text?  Its a comment